### PR TITLE
Fabric java token sdk

### PR DESCRIPTION
**Pull request by smallestant**
This project has the objective to create and provide a java interface to the project [Fabric Token SDK](https://github.com/hyperledger-labs/fabric-token-sdk)

### CODE REVIEW

File: labs/fabric-java-token-sdk.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+title: Lab Proposal Document
+[//]: # (parent: Labs)
```
Comment (tkuhrt) [inline labs/fabric-java-token-sdk.md:4]:
```suggestion
parent: Labs
```

**Review by tkuhrt — CHANGES_REQUESTED** [review]
Please amend the commit history for this PR to include DCO sign-off for each of the commits. The Hyperledger Indy community has [documented steps to fix DCO on previous commits](https://github.com/hyperledger/indy-sdk/blob/main/docs/contributors/signing-commits.md#how-to-sign-previous-commits). Also, the [Fix DCO Guide from src-d](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) contains some different steps you can take.

Can the JNI be added directly to the fabric-token-sdk lab?

File: labs/fabric-java-token-sdk.md (hunk 1 above)
Comment (smallestant) [reply labs/fabric-java-token-sdk.md:4]: Hi, What are the missing commits that were not signed off? 
I only have one under my name and it was signed-off. Or I'm missing something?

Comment (tkuhrt) [reply labs/fabric-java-token-sdk.md:4]: DCO on this PR is failing.

Comment (tkuhrt) [reply labs/fabric-java-token-sdk.md:4]: Based on https://github.com/hyperledger-labs/hyperledger-labs.github.io/pull/253/checks?check_run_id=18539446068, it looks like it is somehow not liking the Signed-off-by message on the commit in this PR. Unfortunately, I cannot tell what it is. Maybe Joao Antunes vs. João Antunes. Not sure if the tilde is causing a problem.

Comment (smallestant) [reply labs/fabric-java-token-sdk.md:4]: I have no idea what I'm doing wrong.
I'm using Intellij to commit. Is it impacting the DCO validation?

**Comment by vipinsun** [comment]
We can merge as administrators without dco because visual inspection reveals the commit is signed off. A bug in the dco script should not stop us!

**Review by lehors — APPROVED** [review]

**Comment by ryjones** [comment]
[here you go, @smallestant ](https://github.com/hyperledger-labs/Fabric-Token-Java-SDK)

**Comment by smallestant** [comment]
Thank you @ryjones

**Pull request merged**